core/common/src/main/java/alluxio/conf/Configuration.java

```diff
@@ -46,24 +59,18 @@
  * <p>
  * This class defines many convenient static methods which delegate to an internal
  * {@link InstancedConfiguration}. To use this global configuration in a method that takes
- * {@link AlluxioConfiguration} as an argument, pass {@link ServerConfiguration#global()}.
+ * {@link AlluxioConfiguration} as an argument, pass {@link Configuration#global()}.
  */
-@NotThreadSafe
-public final class ServerConfiguration {
+@ThreadSafe
```

Comment:
To be verified

Reply:
maybe we can just remove the annotation? It's undefined? 😂